apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -348,12 +348,22 @@ public function get_edit_template_base_url() {
 	 */
 	public function merge_template_and_post( $post ) {
 		// Bail if not a REST API Request.
-		if ( ! ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
+		if ( ! ( defined( 'REST_API_REQUEST' ) && REST_API_REQUEST ) ) {
```

Comment:
@mattwiebe any ideas why REST_REQUEST might have stopped working  here when called from diffs/ endpoint callback? It worked fine during earlier testing but not after a sandbox update and rebase of this PR.